src/Halogen/Aff/Driver/Eval.purs

```diff
@@ -52,23 +53,23 @@ evalF render ref = case _ of
       st { refs = M.alter (const el) p st.refs }
   Action act -> do
     DriverState st <- liftEffect (Ref.read ref)
-    evalM render ref (st.component.eval (Handle act unit))
+    void $ evalM render ref (st.component.eval (Handle act unit))
```

Comment:
Last part of `evalM` is `HalogenM s act ps o Aff ~> Aff` so I guess you don't need `void` here.

Reply:
`evalM` is `-> Aff (Maybe a)` now, so we need this void to avoid a discard error.

Reply:
evalQ is `-> Aff (Maybe a)` but [evalM is `~> Aff`](https://github.com/garyb/purescript-halogen/blob/partial-query/src/Halogen/Aff/Driver/Eval.purs#L68-L73) and as `x ~> y = forall a. x a -> y a` I thought `void` is not needed.

Reply:
Ah yeah, you're right. I must have changed some stuff around while working towards this PR, as I wouldn't have added a `void` without the compiler requiring me to do so :smile:

Reply:
Yeah I thought so 😃